CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
-
+ - decoder API: added `JxlDecoderGetBoxContentsSize` for getting the size of the
```

Comment:
Maybe change to `JxlDecoderGetBoxSizeContents` so the naming pattern follows `JxlDecoderGetBoxSizeRaw`?

Reply:
I have not a strong opinion on this, although perhaps consistency is less important because we plan to deprecate `JxlDecoderGetBoxSizeRaw` in any case.
Wdyt, @veluca93?

Reply:
done, changed the name